src/client/main.ts

```diff
@@ -252,6 +308,9 @@ export function addIntentListener(intent: string, handler: (context: Context) =>
 
 /**
  * Adds a listener for incoming context broadcast from the Desktop Agent.
```

Comment:
Trying to avoid channels - we'll need to go back and do them properly. Agreed on Desktop Agent vs desktop agent. I resolved to lower case, because otherwise I would need to capitalize Intent and Context and I kind of hate that. It looks like a Fighting Fantasy book. You see the Goblin. Will you Test Your Luck, or fight it?